compiler/src/iree/compiler/Codegen/LLVMGPU/test/config_matmul.mlir

```diff
@@ -0,0 +1,213 @@
+// RUN: iree-opt --split-input-file --iree-gpu-test-target=gfx942 --pass-pipeline='builtin.module(linalg-specialize-generic-ops,iree-llvmgpu-select-lowering-strategy)' %s \
+// RUN:   | FileCheck %s --check-prefixes=CHECK,SPECIALIZED
+
+// RUN: iree-opt --split-input-file --iree-gpu-test-target=gfx942 --pass-pipeline='builtin.module(linalg-generalize-named-ops,iree-llvmgpu-select-lowering-strategy)' %s \
+// RUN:   | FileCheck %s --check-prefixes=CHECK,GENERALIZED
+
+
+///////////////////////////////////////////
```

Comment:
nit: mlir uses a different style for section headers, e.g.: https://github.com/llvm/llvm-project/blob/4c389178eeb73618ec80f88d306300949480e40f/mlir/test/Dialect/Vector/vector-contract-to-outerproduct-matvec-transforms.mlir#L168-L170